compiler/rustc_abi/src/extern_abi/mod.rs

```diff
@@ -192,6 +192,10 @@ pub fn is_enabled(
     s
 }
 
+/// Returns whether the ABI is stable to use.
+///
+/// Note that there is a separate check determining whether the ABI is even supported
+/// on the current target; see `fn is_abi_supported` in `rustc_target::spec`.
```

Comment:
Yeah this comment is unrelated, it just did not seem worth its own PR.

Reply:
That's OK